solidity/ecdsa/deploy/02_deploy_reimbursement_pool.ts

```diff
@@ -14,6 +18,11 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
     log: true,
   })
 
+  await deployerSigner.sendTransaction({
+    to: ReimbursementPool.address,
+    value: ethers.utils.parseEther("100.0"), // Send 100.0 ETH
+  })
```

Comment:
Can we do it only if `deployments.getNetworkName() === "hardhat"`?

For mainnet deployment, we definitely not want to send 100 ETH from the deployer account.

Reply:
Yea, good catch, certainly it was added only for the testing hardhat purposes.